doc/rg.1.txt.tpl

```diff
@@ -143,16 +143,16 @@ would behave identically to the following command
 
 same with using globs
 
-    --glob=!git/*
+    --glob=!git
```

Comment:
Ah interesting, yeah, this makes sense to me. We might as well go the whole way here and use `.git` instead of `git`. What do you think?

Reply:
Seems fine! That's what I usually use to exclude `.git` dirs anyway.